### PR TITLE
X 201610 Part 3: libraries

### PR DESCRIPTION
**Pull request by alarcher**

### CODE REVIEW

**Comment by pyhalov** [comment]
can you rename last security patches so that they have appropriate names.

**Comment by alarcher** [comment]
They were not given names yet

**Comment by pyhalov** [comment]
I mean at least something like number(.|-)security-20161004.patch

File: components/x11/libXfont/mapfile-vers [hunk 1]
```diff
@@ -183,13 +183,9 @@ SUNWprivate {
 	pcfWriteFont;
 	pcfError;
 
-	# "src/bitmap/snfstr.h" - needed by xfs
-	SnfSetFormat;
```
Comment (pyhalov) [inline components/x11/libXfont/mapfile-vers:187]: Global symbol disappeared... This is bad, means that we have to rebuild consumers...

Comment (pyhalov) [reply components/x11/libXfont/mapfile-vers:187]: https://lists.x.org/archives/xorg-devel/2014-July/043165.html - We likely don't want libxfont 1.5 unless we ship new Xorg

Comment (alarcher) [reply components/x11/libXfont/mapfile-vers:187]: No no it is not that bad, only xfs, installalias and bdftopcf are affected.
I just saw you merged the other branches, I thought we were waiting a bit for feedback.
I can remove it for now.

Comment (alarcher) [reply components/x11/libXfont/mapfile-vers:187]: So you wanted to merge right now then?
Removing then.

Comment (pyhalov) [reply components/x11/libXfont/mapfile-vers:187]: However, http://upstream.rosalinux.ru/versions/libXfont.html ... But I'd trust @alanc here

Comment (alarcher) [reply components/x11/libXfont/mapfile-vers:187]: Removed, let us update it when we push Xorg 1.18.

Comment (pyhalov) [reply components/x11/libXfont/mapfile-vers:187]: I thought the plan was to merge libraries/applications update in any case and wait feedback before merging xorg and drivers.

Comment (pyhalov) [reply components/x11/libXfont/mapfile-vers:187]: Did you have security update for libxfont?

Comment (alanc) [reply components/x11/libXfont/mapfile-vers:187]: Perhaps the rosalinux site overrode the default --enable flags at compile time?

libXfont 1.5.0 from X.Org changed the default from --enable-snfformat to --disable-snfformat:
- https://lists.x.org/archives/xorg-announce/2014-July/002460.html
- https://cgit.freedesktop.org/xorg/lib/libXfont/commit/?id=2a3429413df27224ceeddd22500ce43b5431d698
- https://cgit.freedesktop.org/xorg/lib/libXfont/commit/?id=37595cfd4feaf031552d66f96dc6d58686f9c851

Comment (alarcher) [inline components/x11/libXfont/mapfile-vers]: @alp There are several CVEs for 1.4.9: CVE-2014-0209, CVE-2014-0210, CVE-2014-0211

Comment (alarcher) [reply components/x11/libXfont/mapfile-vers:187]: To be fixed tonight.

**Pull request merged**